### PR TITLE
Add app control feature on tizen module

### PR DESCRIPTION
**Pull request by daeyeon**
IoT.js-DCO-1.0-Signed-off-by: Daeyeon Jeong daeyeon.jeong@samsung.com

### CODE REVIEW

File: src/iotjs_magic_strings.h [hunk 1]
```diff
@@ -328,6 +328,7 @@
 #endif
 #if ENABLE_MODULE_TIZEN
 #define IOTJS_MAGIC_STRING_TIZEN "tizen"
+#define IOTJS_MAGIC_STRING_EVENT_APP_CONTROL "AppControl"
```
Comment (hs0225) [inline src/iotjs_magic_strings.h]: Please use `#define IOTJS_MAGIC_STRING_APP_CONTROL "AppControl"` to unify with others.

File: src/modules/tizen/iotjs_module_tizen-tizen.c [hunk 2]
```diff
@@ -17,11 +17,138 @@
...
+
+  jerry_value_t tizen = iotjs_module_get(event_emitter_name);
+  jerry_value_t fn = iotjs_jval_get_property(tizen, IOTJS_MAGIC_STRING_EMIT);
+  jerry_value_t jtarget = jerry_create_string((const jerry_char_t*)event_name);
+  jerry_value_t jdata = jerry_create_string((const jerry_char_t*)json);
+
+  // call emit
+  iotjs_jargs_t jargv = iotjs_jargs_create(2);
+  iotjs_jargs_append_jval(&jargv, jtarget);
+  iotjs_jargs_append_jval(&jargv, jdata);
```
Comment (hs0225) [inline src/modules/tizen/iotjs_module_tizen-tizen.c]: You can use `iotjs_jargs_append_string_raw`.

File: test/run_pass/test_tizen_app_control.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+var tizen = require('tizen');
+
+// Need to set target APP_ID e.g) app.control.receiver
+var app_id = '';
+var data = {
+  str: 'iotjs'
+};
+
+// test: tizen.on
+tizen.on('AppControl', function(msg) {
```
Comment (hs0225) [inline test/run_pass/test_tizen_app_control.js]: It seems better to use CamelCase.

**Comment by daeyeon** [comment]
@hs0225 All mentioned are updated.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**